src/main/java/com/treasuredata/client/TDClient.java

```diff
@@ -421,6 +421,15 @@ public void createTable(String databaseName, String tableName)
         doPost(buildUrl("/v3/table/create", databaseName, validateTableName(tableName), TDTableType.LOG.getTypeName()));
     }
 
+    public void createTable(String databaseName, String tableName, boolean includeV)
```

Comment:
Having boolean flag is not ideal because Java has no named arguments unlike Ruby. How about adding methods like `createTableWithV` and `createTableWithoutV`?